app/src/main/java/org/xwiki/android/sync/activities/BaseViewFlipper.java

```diff
@@ -29,22 +30,44 @@
  * BaseViewFlipper
  */
 public abstract class BaseViewFlipper {
+    /**
+     * Activity for operations
```

Comment:
Javadoc recommends to end sentences with a dot.

Reply:
fixed